app/containers/SongList/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ *
+ * SongList
+ *
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components';
+
+// import styled from 'styled-components'
+import { Card, Col, Row, Avatar } from 'antd';
+const CustomCard = styled(Card)`
+
+.songImg{
+  max-height:200px;
+}
+&&{
+   cursor:pointer;
+  margin:10px;
+}
+`
+
+
+function SongList({ songs }) {
+  return (
+    <div data-testid='song-list'>
+      <div className="site-card-wrapper">
+        <Row gutter={10}>
+          {songs.map((songItem) => songItem && songItem.trackName &&
```

Comment:
you can use lodash get instead. 

so `get(songItem, 'trackName') &&`

besides we also have support for optional chaining in the template, so this is legal, `songItem?.trackName`